docs/installation-and-operations/system-requirements/README.md

```diff
@@ -12,10 +12,51 @@ provide any official support for them.
 
 ## Server
 
-### Hardware
+The server hardware requirements should be rougly the same for both the packaged installation and docker (both all-in-one container and compose).
 
+### Minimum Hardware Requirements
+
+* __CPU:__ Quad Core CPU (>= 2ghz)
 * __Memory:__ 4096 MB
-* __Free disk space:__ 4 GB
+* __Free disk space:__ 20 GB
+
+This is for a single Server running OpenProject for up to 200 total active users. Depending on your number of peak users, these requirements might vary drastically.
+
+### General Requirements
+
+Generally speaking you will need more CPUs (the faster the better) and more RAM with an increasing number of users.
+Technically this only really depends on the number of __peak__ users. No matter if you have 1000 or only 100 users overall, if there only ever 20 users working at the same time, the CPU & RAM required will be the same.
+Still, the total number of users is a good general indicator of how much resources you will need.
```

Comment:
I'd add a section that it is not simply a "one worker per concurrent user" equation as almost all pages require multiple requests. So even for one only one concurrent user, I'd recommend having more than one worker available because of that.